packages/kbn-unified-data-table/src/components/row_height_settings.tsx

```diff
@@ -11,13 +11,18 @@ import { i18n } from '@kbn/i18n';
 import { EuiButtonGroup, EuiFormRow, EuiRange, htmlIdGenerator } from '@elastic/eui';
 import { euiThemeVars } from '@kbn/ui-theme';
 
+export enum RowHeight {
+  Auto = 'auto',
+  Single = 'single',
+  Custom = 'custom',
+}
```

Comment:
Or like the following:

```suggestion
export enum RowHeightMode {
  auto = 'auto',
  single = 'single',
  custom = 'custom',
}
```

Like it's implemented is also fine with me. Just a suggestion.